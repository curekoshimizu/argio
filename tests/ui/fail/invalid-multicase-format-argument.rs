#[argio::argio(multicase = "{i + }")]
fn main(n: usize) -> usize {
    n
}
