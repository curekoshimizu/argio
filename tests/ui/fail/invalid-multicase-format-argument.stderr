error: unexpected end of input, expected expression: `i + `
 --> $DIR/invalid-multicase-format-argument.rs:1:28
  |
1 | #[argio::argio(multicase = "{i + }")]
  |                            ^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/invalid-multicase-format-argument.rs:1:1
  |
1 | / #[argio::argio(multicase = "{i + }")]
2 | | fn main(n: usize) -> usize {
3 | |     n
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/ui/fail/invalid-multicase-format-argument.rs`
